nvtabular/columns/schema_io/schema_writer_base.py

```diff
@@ -0,0 +1,12 @@
+class SchemaWriter:
+    @classmethod
+    def _read_scheam(cls, schema_path):
+        raise NotImplementedError("Must have logic to read schema from file")
+
+    @classmethod
+    def write_schema(cls, schema, schema_path):
```

Comment:
Since the class names already have `Schema` in them, we can probably drop `_schema` from these method names so they're just `write`,`load`, and `_read`